packages/core-web/src/styles/markbind.css

```diff
@@ -142,6 +142,18 @@ footer {
     color: #777;
 }
 
+.large {
+    font-size: 125%;
+}
+
+.small {
+    font-size: 75%;
+}
```

Comment:
Just a minor opinion here, I think it would be somewhat satisfying (not sure if it is "better" though, feel free to discuss) to have 80% percentage for small. That way, `++--text--++` will have exactly the same size as plain `text` as the percentages would be reciprocal 😄 

The percentage difference from the normal size against larger/smaller size would be unequal, but 5% difference from 75% could also be deemed small enough to be negligible. Again, this is pretty much subjective, so it is up to you (and others if anyone wants to weigh in) =)

Reply:
agree here, its also consistent with bootstrap's reboot css sizing for the html `<small>` tag

Reply:
Thanks for the suggestion! @ryoarmanda :-)